### ISSUE TITLE
Strider built-in layout panics on Ctrl+D in basic focused pane

### ISSUE THREAD

**Issue by alekspickle** [zellij-org/zellij#2015]
`zellij --version`: 0.34.3
`stty size`: 33 110
`uname -av` or `ver`(Windows): Linux pickle 5.15.0-56-generic #62-Ubuntu SMP Tue Nov 22 19:54:14 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

`alacritty --version`: alacritty 0.10.1

**Further information**
`zellij -l strider`
Press Ctrl+D twice. Ideally it should just close focused pane, but it crashes instead.
The reason seems to be 
`zellij-server-0.34.3/src/panes/grid.rs:2921]: Unhandled esc_dispatch: 61->`

but I'll add logs anyway:
<details>
<summary>log</summary>

```


INFO   |zellij_client            | 2022-12-12 13:12:47.657 [main      ] [/home/pickle/.cargo/registry/src/github.com-1ecc6299db9ec8
      23/zellij-client-0.34.3/src/lib.rs:140]: Starting Zellij client!
    1 INFO   |zellij_server            | 2022-12-12 13:12:47.660 [main      ] [/home/pickle/.cargo/registry/src/github.com-1ecc6299db9ec8
      23/zellij-server-0.34.3/src/lib.rs:213]: Starting Zellij server!
    2 INFO   |zellij_server::plugins   | 2022-12-12 13:12:47.712 [wasm      ] [/home/pickle/.cargo/registry/src/github.com-1ecc6299db9ec8
      23/zellij-server-0.34.3/src/plugins/mod.rs:61]: Wasm main thread starts
    3 INFO   |zellij_client::input_hand| 2022-12-12 13:12:47.713 [input_handler] [/home/pickle/.cargo/registry/src/github.com-1ecc6299db9
      ec823/zellij-client-0.34.3/src/input_handler.rs:151]: sending done loading
    4 INFO   |zellij_client::input_hand| 2022-12-12 13:12:47.713 [input_handler] [/home/pickle/.cargo/registry/src/github.com-1ecc6299db9
      ec823/zellij-client-0.34.3/src/input_handler.rs:155]: done sent done loading
    5 INFO   |zellij_utils::input::plug| 2022-12-12 13:12:47.714 [wasm      ] [/home/pickle/.cargo/registry/src/github.com-1ecc6299db9ec8
      23/zellij-utils-0.34.3/src/input/plugins.rs:139]: Plugin 'tab-bar.wasm' exists in the 'PLUGIN DIR' at '/home/pickle/.local/share/ze
      llij/plugins/' but is being ignored
    6 INFO   |zellij_utils::input::plug| 2022-12-12 13:12:47.721 [wasm      ] [/home/pickle/.cargo/registry/src/github.com-1ecc6299db9ec8
      23/zellij-utils-0.34.3/src/input/plugins.rs:139]: Plugin 'strider.wasm' exists in the 'PLUGIN DIR' at '/home/pickle/.local/share/ze
      llij/plugins/' but is being ignored
    7 INFO   |zellij_utils::input::plug| 2022-12-12 13:12:47.786 [wasm      ] [/home/pickle/.cargo/registry/src/github.com-1ecc6299db9ec8
      23/zellij-utils-0.34.3/src/input/plugins.rs:139]: Plugin 'status-bar.wasm' exists in the 'PLUGIN DIR' at '/home/pickle/.local/share
      /zellij/plugins/' but is being ignored
    8 INFO   |zellij_client::input_hand| 2022-12-12 13:12:48.214 [input_handler] [/home/pickle/.cargo/registry/src/github.com-1ecc6299db9
      ec823/zellij-client-0.34.3/src/input_handler.rs:158]: sending done l


```

</details>

**Comment by naosense** [comment]
I can reproduct it.
```
Error occurred in server:

  × Thread 'screen' panicked.
  ├─▶ Originating Thread(s)
  │   	1. stdin_handler_thread: AcceptInput
  │   	2. screen_thread: WriteCharacter
  │
  ├─▶ At zellij-server/src/lib.rs:699:18
  ╰─▶ Program terminates: a fatal error occured

      Caused by:
          failed to write to active terminal for client 1 - msg: [4]

      Stack backtrace:
         0: __mh_execute_header
         1: __mh_execute_header
         2: __mh_execute_header
         3: __mh_execute_header
         4: __mh_execute_header
         5: __mh_execute_header
         6: __pthread_deallocate

      Panic backtrace:
         0: __mh_execute_header
         1: __mh_execute_header
         2: __mh_execute_header
         3: __mh_execute_header
         4: __mh_execute_header
         5: __mh_execute_header
         6: __mh_execute_header
         7: __mh_execute_header
         8: __mh_execute_header
         9: __mh_execute_header
        10: __mh_execute_header
        11: __mh_execute_header
        12: __mh_execute_header
        13: __mh_execute_header
        14: __pthread_deallocate

  help: If you are seeing this message, it means that something went wrong.
        Please report this error to the github issue.
        (https://github.com/zellij-org/zellij/issues)

        Also, if you want to see the backtrace, you can set the `RUST_BACKTRACE` environment variable to `1`
```

**Comment by imsnif** [comment]
ping @har7an  - I've seen this too, I'm pretty sure this has to do with the resizing change. I wrote more details in Matrix. This is very high priority, so if you can take a look ASAP it would be great. Thanks!

**Comment by har7an** [comment]
Hello everyone,

I'm sorry you're seeing this behavior! A fix for it (#2019) is already on the way to main and will be part of the next release.
